### DISCUSSION TITLE
p:selectCheckboxMenu group selection

### DISCUSSION THREAD

**Discussion by prancius** [primefaces/community#565] [PrimeFaces] [answered]
p:selectCheckboxMenu it is possible to show grouped data. 
How to enable group selection instead of clicking one by one item?

**Comment by melloware** [comment, marked as answer]
It has grouping you can see here: https://www.primefaces.org/showcase/ui/input/checkboxMenu.xhtml   But it does not allow you to select a group and check its boxes.

The issue is currently open already on GitHub Issues: https://github.com/primefaces/primefaces/issues/3524
